chain-extensions/types/xvm/src/lib.rs

```diff
@@ -41,8 +41,7 @@
             SameVmCallNotAllowed => 2,
             InvalidTarget => 3,
             InputTooLarge => 4,
-            BadOrigin => 5,
-            ExecutionFailed(_) => 6,
+            ExecutionFailed(_) => 5,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `ExecutionFailed` variant is missing in the match arm. Please add it back.
```diff
+            ExecutionFailed(_) => 5,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->